src/ugeneui/src/main_window/MainWindowImpl.h

```diff
@@ -36,6 +36,7 @@ class QToolBar;
 #if defined(Q_OS_DARWIN) && !defined(_DEBUG)
 #    define _INSTALL_TO_PATH_ACTION
 #endif
+#undef _INSTALL_TO_PATH_ACTION
```

Comment:
Why we define this 2 lives above and undefine it later?

Reply:
Oh, will check

Reply:
UGENE-7195
Need to update menu item which creates the link to the app on desktop (MacOS)